parser/parser.go

```diff
@@ -486,6 +486,10 @@ func (p *parser) unquote(ctx interface{}, value string) string {
 	if text, err := strconv.Unquote(value); err == nil {
 		return text
 	}
+	// Handle the unquoting of single quotes.
+	if len(value) >= 2 && value[0] == '\'' && value[len(value)-1] == '\'' {
+		return value[1:len(value)-1]
+	}
```

Comment:
Add comments for whatever remains to be done for string parsing, e.g. escapes, raw strings, triple-quoted, etc.

Add a unit test in parser_test.